File: c/Vetores/numeros_pares.c
#include <stdio.h>

int main() {

    int n;
    int numeros[100];
    int quantidadePares = 0;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        printf("Digite um numero: ");
        scanf("%d", &numeros[i]);
    }

    printf("\nNUMEROS PARES:\n");

    for (int i = 0; i < n; i++) {
        if (numeros[i] % 2 == 0) {
            printf("%d ", numeros[i]);
            quantidadePares++;
        }
    }

    printf("\n\nQUANTIDADE DE PARES = %d\n", quantidadePares);

    return 0;
}

File: java/Vetores/NumerosPares.java
import java.util.Scanner;

public class NumerosPares {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int N;

        System.out.print("Quantos numeros serao digitados: ");
        N = sc.nextInt();

        int[] numeros = new int[N];

        for (int i = 0; i < N; i++) {
            System.out.print("Digite um numero: ");
            numeros[i] = sc.nextInt();
        }

        int quantidadePares = 0;

        System.out.println("Numeros pares:");

        for (int i = 0; i < N; i++) {

            if (numeros[i] % 2 == 0) {
                System.out.println(numeros[i]);
                quantidadePares++;
            }
        }

        System.out.println("Quantidade de numeros pares: " + quantidadePares);

        sc.close();
    }
}